metricbeat/module/windows/perfmon/reader.go

```diff
@@ -94,21 +95,20 @@ func NewReader(config Config) (*Reader, error) {
 			r.measurement[v] = counter.MeasurementLabel
 		}
 	}
-
 	return r, nil
 }
 
 // RefreshCounterPaths will recheck for any new instances and add them to the counter list
-func (r *Reader) RefreshCounterPaths() error {
+func (this *Reader) RefreshCounterPaths() error {
```

Comment:
Oh, memories of java world :slightly_smiling_face:

Reply:
all changed